packages/public/server/src/module/Taxonomy/src/Manager/TaxonomyManager.php

```diff
@@ -377,6 +377,18 @@ public function updateTerm(FormInterface $form)
         return $term;
     }
 
+    public function flushUpdatesForTerm(TaxonomyTermInterface $term)
+    {
+        $this->flush();
+        // We need to detach $term from the ORM cache and reload a new $term from the database or otherwise the
+        // new children order is not stored
+        $this->objectManager->detach($term);
+        $updatedTerm = $this->getTerm($term->getId());
+        $this->getEventManager()->trigger('update', $this, [
+            'term' => $updatedTerm,
+        ]);
```

Comment:
Alternative: We can move this logic to `updateTerm()` for the parent element which is updated. However then we trigger multiple update events for the parent element.